crates/burn-tensor/src/tensor/api/numeric.rs

```diff
@@ -357,6 +357,21 @@ where
         (tensor, index)
     }
 
+    /// Find the maximum pair wise values with another Tensor
```

Comment:
Find => Finds

Reply:
Singular `Find` was inline with other comments. Do you want existing ones changed to plural as well?

Reply:
It's a description that should read as [This function] Finds ... 

Others should follow the same but you can leave other functions as is for now.